spec/sidekiq/jobs/inning_rollover_spec.rb

```diff
@@ -43,6 +43,12 @@
       expect(@inning2.turing_modules.where(module_number: 6).count).to eq(1)
     end
 
+    it 'removes all zoom_aliases from prior modules' do
+      InningRolloverJob.perform_async(@inning2.id)
```

Comment:
I think we should confirm if any zoom_aliases exist or manually create the zoom_aliases before this test to ensure that the functionality is working correctly.